wallet/types/src/account_id.rs

```diff
@@ -63,6 +67,7 @@ impl<Id: Encode> AccountPrefixedId<Id> {
 
 impl<Id: Encode> HasPrefix<AccountId> for AccountPrefixedId<Id> {}
 
+pub type AccountWalletSTxId = AccountPrefixedId<Id<Transaction>>;
```

Comment:
That's some weird capitalization of letters... is that S a typo?

Reply:
renamed